### ISSUE TITLE
openbabel install broken on some systems; can be fixed with libxrender dependency

### ISSUE THREAD

**Issue by ianrinehart** [CCEMGroupTCD/DARTassembler#1]
**tl;dr:** 

Just wanted to be helpful, as the first step to installing this didn't go smoothly on my system, but I found a quick fix. 

Not all systems will have libxrender installed by default. To make the installation of this package work more generally, the pip install of openbabel should be augmented with the optional dependency libxrender. The solution is to run:

sudo apt-get install libxrender1

before running the "DARTassembler installtest --path" command

...and then everything works. 

**Details**

OS: Ubuntu 24.04 on WSL2
Installation following docs @ https://dartassembler.readthedocs.io/en/latest/doc_files/1_installation_guide.html, using pip inside of a micromamba environment

During the installtest commandline operation, a series of errors arise which look like this (one for every single .so file in the openbabel installation):

```shell
==============================
*** Open Babel Error  in openLib
  /home/{user}/micromamba/envs/ldrd/lib/python3.12/site-packages/openbabel/lib/openbabel/3.1.0/groformat.so did not load properly.
 Error: libXrender.so.1: cannot open shared object file: No such file or directory
==============================
``` 

Per openbabel [documentation](https://openbabel.org/docs/Installation/install.html), libxrender is an "optional" dependency, but not having it makes loading the shared object files for a pip openbabel install fail, causing the error seen above. This is a limitation of the pip install of openbabel. 

This is different from the error mentioned briefly in the dev/ folder README on this repository (that read/write permissions may be an issue). Updating readwrite permissions does not affect this error, installing openbabel separately using pip outside of a micromamba environment does not fix the issue, and using sudo at any time does not affect this issue. The open babel error is not very descriptive, but it can arise if libxrender is not installed.

**Comment by CCEMGroupTCD** [comment]
Hey Ian,

great catch! I will update the documentation later today and close this issue then. 

Thank you very much for the detailed report and the troubleshooting. If there is anything else that you notice about DART, please drop us another issue!

**Comment by CCEMGroupTCD** [comment]
Closing this issue with commit 582d0d0eb6be7f277f779ae32c79e44daf69550b.

Thanks for your great help Ian! We added installation of the library "libxrender1" to the documentation (not "libxrender", this seems to have been a typo"). Please let us know if you encounter anything else.
